uber/models.py

```diff
@@ -448,16 +445,32 @@ def initialize_db(cls, modify_tables=False, drop=False):
         all models from all plugins to insert their mixin data, so we wait until one spot
         in order to create the database tables.
 
-        Any calls to initialize_db() that do not specify modify_tables=True are ignored.
-        i.e. anywhere in Sideboard that calls initialize_db() will be ignored
-        i.e. ubersystem is forcing all calls that don't specify modify_tables=True to be ignored
+        Any calls to initialize_db() that do not specify modify_tables=True or
+        drop=True are ignored.
+
+        i.e. anywhere in Sideboard that calls initialize_db() will be ignored.
+        i.e. ubersystem is forcing all calls that don't specify modify_tables=True
+        or drop=True to be ignored.
+
+        Calling initialize_db with modify_tables=False and drop=True will leave
+        you with an empty database.
 
         Keyword Arguments:
-        modify_tables -- If False, this function does nothing.
-        drop -- USE WITH CAUTION: If True, then we will drop any tables in the database
+            modify_tables: If False, this function will not attempt to create
+                any database objects (tables, columns, constraints, etc...)
+                Defaults to False.
+            drop: USE WITH CAUTION: If True, then we will drop any tables in
+                the database. Defaults to False.
         """
         if modify_tables:
-            super(Session, cls).initialize_db(drop=drop)
+            super(Session, cls).initialize_db(drop=drop, create=True)
+            if drop:
+                from uber.migration import stamp
+                stamp('heads')
```

Comment:
Nice.  Although in general I'm guessing we won't want to do that except in weird testing situations, right?  Like, now that we have migrations that can create the database from scratch, we should probably be using them all the time instead of relying on Sideboard's "automatically create all my tables for me" behavior.